### PR TITLE
WIP[feat]: product page

### PR DESCRIPTION
**Pull request by saadahmadhere**
- add markup and style for the product page.

https://user-images.githubusercontent.com/52596800/154863858-01a07feb-a5ee-4b7d-a9e1-588534b38e38.mp4

### CODE REVIEW

File: css/product.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: var(--size-6);
+  margin-bottom: var(--size-4);
+}
+
+.filter {
+  justify-content: space-between;
+}
```
Comment (nareshpoonia) [inline css/product.css:50]: You can consider making a justify-between utility class for the same

**Review by FarhanMobashir — APPROVED** [review]
everything is neat and good to go. can be merged

File: css/product.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+/* products */
+.product_title {
+  margin: var(--size-4) var(--size-1) var(--size-8) var(--size-4);
+}
+
+/* footer */
+#footer {
+  background-color: var(--gray-300);
+  display: flex;
+  flex-direction: column;
```
Comment (nareshpoonia) [inline css/product.css]: Can considering using flex-col utility class

File: pages/product.html [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>nftcart | products</title>
+
+    <!-- Favicon -->
+    <link rel="icon" type="image/x-icon" href="../assets/nft.ico" />
+
+    <!-- Fontawesome -->
+    <script
+      src="https://kit.fontawesome.com/b9c207240a.js"
+      crossorigin="anonymous"></script>
```
Comment (nareshpoonia) [inline pages/product.html:15]: You can consider moving font-awesome script to main css, so that you don't have to replicate it to all of your html files

**Review by Shivanshsharma01 — APPROVED** [review]
The code looks clean. No changes from my side

**Review by nareshpoonia — APPROVED** [review]
Just a few small comments, otherwise code looks clean.
Good to merge

**Review by rahul0396 — APPROVED** [review]
Clean and optimized code. Good to be merged

File: css/product.css (hunk 1 above)
Comment (saadahmadhere) [reply css/product.css:50]: Roger that

File: css/product.css (hunk 2 above)
Comment (saadahmadhere) [reply css/product.css]: done!

File: pages/product.html (hunk 3 above)
Comment (saadahmadhere) [reply pages/product.html:15]: Yes, this just for momentary basis.

**Pull request merged**